dlt-connector/src/graphql/resolver/CommunityResolver.ts

```diff
@@ -0,0 +1,69 @@
+import { Resolver, Query, Arg, Mutation } from 'type-graphql'
+
+import { CommunityDraft } from '@input/CommunityDraft'
+
+import { getDataSource } from '@typeorm/DataSource'
+
+import { TransactionResult } from '../model/TransactionResult'
+import { TransactionError } from '../model/TransactionError'
+import {
+  create as createCommunity,
+  iotaTopicFromCommunityUUID,
+  isExist,
+} from '@/controller/Community'
+import { TransactionErrorType } from '../enum/TransactionErrorType'
+import { logger } from '@/server/logger'
+
+@Resolver()
+export class CommunityResolver {
+  // Why a dummy function?
+  // to prevent this error by start:
+  //   GeneratingSchemaError: Some errors occurred while generating GraphQL schema:
+  //     Type Query must define one or more fields.
+  // it seems that at least one query must be defined
+  // https://github.com/ardatan/graphql-tools/issues/764
+  @Query(() => String)
+  version(): string {
+    return '0.1'
+  }
```

Comment:
The dummy query is necessary if there is no other query. No matter how many mutations there are. However, a query in one of the resolvers is sufficient, there does not need to be a query in every resolver. I just tried this, I can remove the dummy query from the community resolver because there is still one in the transaction resolver.